spec/controllers/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET users' do
+    before(:each) do
+      @user = User.new(name: 'Tom', postscounter: 0)
+      @user.save
+    end
+
+    it 'renders the index template' do
+      get '/users'
+      expect(response.status).to eq(200)
+      expect(response).to render_template('index')
+    end
+
+    it 'renders the index template' do
+      get '/users/1'
+      expect(response.status).to eq(200)
+      expect(response).to render_template('show')
```

Comment:
* Kindly add tests for your show actions as stated in the [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Make%20sure%20that%20for,includes%20correct%20placeholder%20text.)